### ISSUE TITLE
vitest/no-conditional-tests - false positive

### ISSUE THREAD

**Issue by youleapnir** [veritem/eslint-plugin-vitest#18]
Hi
I'm using the `vitest/no-conditional-tests` rule, but it alerts all my tests.
 Here is an example:
```tsx
function myFunc(str: string) {
  return str;
}

describe("myTest", () => {
  it("convert shortened equal filter", () => {
    expect(
      myFunc("5")
    ).toEqual("5");
  });
});
```

I expect this to be ok.
Thanks!
